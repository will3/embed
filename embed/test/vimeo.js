const test = require('./utils/test');

test('vimeo',
  'https://vimeo.com/216098603');
