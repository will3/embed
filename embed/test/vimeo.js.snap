// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` vimeo 1`] = `
Object {
  "description": "Dreamy,  Day of the Dead Mexican textile animation for Husky&#039;s track Ghost. Taken from their upcoming new album Punchbuzz.  Ghost is available now - http://bit.ly/2oW45pm  Directed&hellip;",
  "originalUrl": "https://vimeo.com/216098603",
  "site_name": "Vimeo",
  "thumbnail": Object {
    "url": "https://i.vimeocdn.com/video/632985003_1280x720.jpg",
  },
  "title": "HUSKY - GHOST on Vimeo",
  "video": Object {
    "description": undefined,
    "height": "720",
    "title": undefined,
    "url": "https://player.vimeo.com/video/216098603",
    "width": "1280",
  },
}
`;
